integration_tests/test_dict_keys_values.py

```diff
@@ -51,4 +51,27 @@ def test_dict_keys_values():
                 assert v2_copy[j] == d2[str(i)]
         assert key_count == 1
 
+
+    # dict.keys on dict constant
+    assert len({1: "a"}.keys()) == 1
```

Comment:
Can you update the tests according to https://github.com/lcompilers/lpython/pull/2661#discussion_r1580436211?

Reply:
I had initially tried this @Shaikh-Ubaid , but it throws the error as stated by you in #2667. I am looking into it.